app/templates/restore_events.html

```diff
@@ -0,0 +1,58 @@
+{% extends "base.html" %}
+
+{% block head %}
+    {{ super() }}
+{% endblock %}
+
+{% block content %}
+    <h4>Restore deleted events</h4>
+    <form action="restore_events" method="post">
+        <table class="table">
+            <thead>
+            <tr>
+                <th scope="col">#</th>
+                <th scope="col">Event ID</th>
+                <th scope="col">Title</th>
+                <th scope="col">Start Event</th>
+                <th scope="col">End Event</th>
```

Comment:
what about the contents?

Reply:
I do not think it is necessary now, I can add it after :)